### PR TITLE
fix(fdb): rectify bad rebase MTA-3490b

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Comment by SamyOubouaziz** [comment]
🙌

File: managed-databases/document-database/reference-content/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+FerretDB translates MongoDB® wire protocol queries to SQL to be remain compatible with existing software and tools built on MongoDB®.
+
+<Lightbox src="scaleway-ferretdb-schema.webp" alt="" />
+
+The image above depicts a common use-case of Scaleway Managed Document Databases. Managed Document Databases are hosted on Instances running a PostgreSQL Agent and a FerretDB container.
+
+In this example, the Database Instance is attached to a Private Network. High Availability is also activated, meaning that data is streamed continuously from the Database Instance to a secondary node. In the examlpe, Database Instance configuration is stored on a specific distributed storage, and [heartbeats](/managed-databases/document-database/concepts#heartbeats) are continuously performed to monitor the Instance's status and redirect traffic automatically if an incident is detected on the primary Database Instance. Finally, the Database Instance data is stored on Block Storage.
```
Comment (fpagny) [inline managed-databases/document-database/reference-content/overview.mdx:25]: Typo on example.
```suggestion
In this example, the Database Instance is attached to a Private Network. High Availability is also activated, meaning that data is streamed continuously from the Database Instance to a secondary node. In the example, Database Instance configuration is stored on a specific distributed storage, and [heartbeats](/managed-databases/document-database/concepts#heartbeats) are continuously performed to monitor the Instance's status and redirect traffic automatically if an incident is detected on the primary Database Instance. Finally, the Database Instance data is stored on Block Storage.
```

File: managed-databases/document-database/reference-content/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+During the Public Beta phase of Managed Document Databases, you can contribute to FerretDB's open-source project by anonymously sharing the telemetry data of your FerretDB Database Instances.
+
+### Collected data
+
+The data that is collected is anonymous, and is restrained to:
+
+  - FerretDB version
+  - random instance UUID
+  - autonomous system number, cloud provider region, or country derived from IP address (but the IP address itself)
```
Comment (fpagny) [inline managed-databases/document-database/reference-content/overview.mdx:37]: Add "but NOT the IP address itself".
```suggestion
  - autonomous system number, cloud provider region, or country derived from IP address (but the not IP address itself)
```

File: managed-databases/document-database/reference-content/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - uptime
+  - backend (PostgreSQL or SQLite) version
+  - installation type (Docker, package, cloud provider marketplace, self-built)
+  - build configuration (Go version, build flags and tags)
+  - command statistics:
+    - protocol operation codes (e.g. OP_MSG, OP_QUERY);
+    - command names (e.g. find, aggregate);
+    - arguments (e.g. sort, $count (stage));
```
Comment (fpagny) [inline managed-databases/document-database/reference-content/overview.mdx:45]:
```suggestion
    - arguments (e.g. sort, $count (stage), but not parameters values or response contents);
```

**Pull request merged**